src/Uno.UI/UI/Xaml/ResourceDictionary.cs

```diff
@@ -21,6 +21,30 @@ public partial class ResourceDictionary : DependencyObject, IDependencyObjectPar
 		private readonly List<ResourceDictionary> _mergedDictionaries = new List<ResourceDictionary>();
 		private ResourceDictionary _themeDictionaries;
 
+		/// <summary>
+		/// This event is fired when a key that has value of type <see cref="ResourceDictionary"/> is added or changed in the current <see cref="ResourceDictionary" />
+		/// </summary>
+		private event EventHandler<ResourceDictionaryChangedEventArgs> ResourceDictionaryValueChange;
+
+		private class ResourceDictionaryChangedEventArgs
```

Comment:
This class is no longer needed since we invalidate the cache for any changes now. But it may (or may not) be used later. I'm not sure if I should remove it or keep it. @davidjohnoliver @jeromelaban What do you think?

Reply:
Personally I would remove it if it's not used, it's internal so there's no difficulty in bringing it back if it becomes necessary.